api/api.proto

```diff
@@ -514,14 +516,29 @@ message SnapCreateResponse {
   VolumeCreateResponse volume_create_response = 1;
 }
 
-// VolumeInfo 
+// VolumeInfo
 // swagger:model
 message VolumeInfo {
   string volume_id = 1;
   string path = 2;
   VolumeSpec storage = 3;
 }
 
+// VolumeConsumer
+// swagger: model
+message VolumeConsumer {
```

Comment:
I kept it as VolumeConsumer since I didn't want to put an explicit restriction. The primary use case would be to plug in consumer's info from an orchestrator perspective. Keeping the struct generic will help plug in consumer even when there is no orchestrator. For e.g just a `docker run -v src:dest` or a use of a debug CLI from the volume driver to consume the volume (for e.g for Portwox, this can be pxctl host attach and mount from a node.)